functions/controllers/post/newPost.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable max-len */
+// const firebase = require('firebase');
+const HttpStatus = require('http-status-codes');
+// const { validateCode } = require('../../util/validator');
+const { status, message } = require('../../util/constants');
+
+const { error, success } = status;
+const {
+    somethingWentWrong,
+} = message;
+const {
+    INTERNAL_SERVER_ERROR,
```

Comment:
const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');
preferred approach now

Reply:
fixing the http status code import would fix the conflict errors..as they have been implemented on my end on master branch